### DISCUSSION TITLE
Page Refreshing when I click on Dashboard

### DISCUSSION THREAD

**Discussion by praveenvall** [LiveHelperChat/livehelperchat#2068] [General]
Hello, 
I am using the latest version 4.32 of LHC.
For the last two versions, the dashboard is refreshing completely instead of onpage switching between tabs.
This is forcing to refresh the entire dashboard for chat tool and delay occurs while answering to the users.
This also happens, if we try to answer the second chat if any, while switching the chats.
Please help!

**Comment by remdex** [comment]
I ques you missed something during update. Or reproduce on demo and write down a steps to reproduce.

**Reply by praveenvall** [reply]
I am sorry for the delay. Here is an [youtube video](https://youtu.be/AmUTTnw9KDw) which shows the issues encountering during the updates.

**Reply by praveenvall** [reply]
[second link ](https://www.youtube.com/watch?v=i1S3v0552ic)of procedure, I always follow to replace the old files from file manager

**Reply by remdex** [reply]
Sorry, but there is something not right with your setup... You can try to write me in private chat...
